NoSoft-SA/nspack
# Feature/manual edi improvements

Summary of this change:
Quality of life changes and bugfixes for EDI in transactions, specifically manual intakes.

CHANGELOG entry:

### Added
- `Clone` action menu item to validate- and manual intake screens which clones the given row (minus the sscc and seq no.).
- `View manual intake` action menu item to EDI in transactions (only applicable to `manual_process` POs).
### Fixed
- Removed n/a action menu items to manual intakes that caused error emails.

How to test this code via the UI:
Create a manual intake and confirm that rows can be cloned. Process it and ensure no validation is required. Finally view it.
